client/src/components/pages/Posts/index.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { Link, BrowserRouter as Router } from 'react-router-dom';
+import { Icon } from 'antd';
+import Post from '../../utils/post';
+import './style.css';
+
+export default function Posts() {
+  return (
+    <Router>
```

Comment:
@IsraaSulaiman I think you might not need a `<Router>` here since there's no `<switch>` inside it.  You should still be able to use `<Link>` without the `<Router>` here, hopefully.

Reply:
I had to wrap the Links within a Router component here because we have not yet made our react routes. `<Link>` won't work in this case without `<Router>` since it needs the history prop, I guess.

So it is temporary. I'll remove it once we create our routes in the app.

Reply:
Oh ok, sounds good. Thanks for explaining that to me!